setup.cfg

```diff
@@ -27,8 +27,6 @@ python_files =
     test_*.py
     *_test.py
     tests.py
-json_report = report.json
-jsonapi = true
```

Comment:
Were these two changes meant to be included in this?

Reply:
Yes, because pytest complained/printed warnings when they were present:

https://github.com/napalm-automation/napalm/actions/runs/9962394420/job/27526038035#step:8:729

Reply:
And I believe we pass these as command line options in the docs build: https://github.com/napalm-automation/napalm/blob/develop/docs/test.sh#L8